modules/nomad-clients/user_data.sh

```diff
@@ -0,0 +1 @@
+../core/user_data/user-data-nomad-client.sh
```

Comment:
This is a symlink.

Reply:
Noted, terraforming on Windows will have a harder time lol :smile: